packages/paste-theme/src/types/themes.ts

```diff
@@ -1,9 +1,7 @@
 import type {DefaultTheme} from '../themes/default';
 import type {SendGridTheme} from '../themes/sendgrid';
-import type {ConsoleTheme} from '../themes/console';
 import type {DarkTheme} from '../themes/dark';
 
 export type SendGridThemeShape = typeof SendGridTheme;
-export type ConsoleThemeShape = typeof ConsoleTheme;
```

Comment:
So removing Console things from the Theme package _is_ a breaking change. But we want this to be seemless. Just as how we're pointing `theme="console"` to use the `default` theme, we can do things like
```
export type ThemeShape = typeof DefaultTheme;
export type ConsoleThemeShape = ThemeShape;
```

So we don't import any console stuff, but we continue to export it as default.

Reply:
This is a good point, and also affects the theme/inde.ts file too. https://github.com/twilio-labs/paste/pull/2168/files#diff-e12cea4607d991859b3334966fe268a7b9c542d9d643607f2422787dab61618bL3

We export the Console theme from the theme package, so we should also remap that export to the Default theme, so to the consumer, we still `export {DefaultTheme, ConsoleTheme, SendGridTheme, DarkTheme};`

Reply:
Ok! I added the exports back in this file and the `paste-theme/src/themes/index.ts`